app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const { getCategories } = require("./controllers/categories");
+const { getReviews } = require("./controllers/reviews");
+const app = express();
+
+app.use(express.json());

```

Comment:
This middleware should not be included. You should remove line 6

Reply:
I removed it sorry its because i did another pr too quick before the first got approved